### PR TITLE
demos/tests: add smart_classroom

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: demos/multichannel_demo/models.lst [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # This file can be used with the --list option of the model downloader.
 face-detection-retail-????
+face-detection-adas-????
+face-detection-adas-binary-????
```
Comment (IRDonch) [inline demos/multichannel_demo/models.lst]: Keep these in sorted order, please.

File: demos/tests/image_sequences.py [hunk 2]
```diff
@@ -135,4 +135,17 @@
         image_net_arg('00037128'),
         image_net_arg('00048316'),
     ],
+
+    '500x375x3': [
```
Comment (IRDonch) [inline demos/tests/image_sequences.py]: Is this just a sequence of arbitrary same-sized images? I don't think we're going to get good code coverage with a sequence like that - you can't reidentify faces if there aren't any faces.

File: demos/multichannel_demo/models.lst (hunk 1 above)
Comment (Wovchena) [reply demos/multichannel_demo/models.lst]: Updated

File: demos/tests/image_sequences.py (hunk 2 above)
Comment (Wovchena) [reply demos/tests/image_sequences.py]: Updated images

**Comment by vladimir-dudnik** [comment]
looks good to me

**Pull request merged**